src/mprk.jl

```diff
@@ -24,19 +24,29 @@ function build_mprk_matrix(P, sigma, dt)
 end
 
 # in-place for dense arrays
-function build_mprk_matrix!(M, P, sigma, dt)
+function build_mprk_matrix!(M, P, sigma, dt; d = nothing)
```

Comment:
I tried to reuse as much code as possible, so added `d` which stores the non-conservative destruction terms as an optional argument.

Reply:
I think it's fine - but I would use `d` as optional positional argument, not as keyword argument, i.e.,
```suggestion
function build_mprk_matrix!(M, P, sigma, dt, d = nothing)
```

Reply:
Done.